addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -71,6 +71,13 @@ def _check_move_configuration(self, invoice):
         if self.code != 'es_tbai' or invoice.country_code != 'ES':
             return errors
 
+        if invoice.is_purchase_document():
+            if not invoice.ref:
+                errors.append(_("You need to fill in the Reference field as the invoice number from your vendor."))
+            elif not any(x.isdecimal() for x in invoice.ref):
+                errors.append(_("As the Reference field should have the invoice number from your vendor, "
+                                "it should contain digits."))
```

Comment:
IIUC, this is not necessary anymore, now that we're not trying to extract the sequence and number from it.

```suggestion
```

Reply:
I think the number of the vendor should still contain a number, so it would still be a valid check.